src/main/java/com/iexec/standalone/metric/MetricService.java

```diff
@@ -1,61 +0,0 @@
-/*
- * Copyright 2020 IEXEC BLOCKCHAIN TECH
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.iexec.standalone.metric;
-
-import com.iexec.standalone.chain.DealWatcherService;
-import com.iexec.standalone.task.TaskStatus;
-import com.iexec.standalone.task.event.TaskStatusesCountUpdatedEvent;
-import com.iexec.standalone.worker.WorkerService;
-import org.springframework.context.event.EventListener;
-import org.springframework.stereotype.Service;
-
-import java.util.LinkedHashMap;
-
-@Service
-public class MetricService {
```

Comment:
I think this removal is a bit too strong.
We may be interested to know if the standalone worker is full or not and follow the number of tasks per status on the `/metrics` endpoint ?
Do we want to remove this or try to keep this and we need to study how to modify this class ?
If something has already been decided regarding this, I will follow the decision